tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -88,6 +89,22 @@ export class RunsDataTable {
     );
   }
 
+  selectionClick(event: MouseEvent, runId: string) {
+    // Prevent checkbox from switching checked state on its own.
+    event.preventDefault();
```

Comment:
If I recall correctly, I attempted this approach for the original implementation but couldn't get it to work properly. I would guess I was missing this call to preventDefault().

Nice work.

Reply:
Thanks